metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -251,7 +251,7 @@ class MetalsLanguageServer(
   private var symbolSearch: MetalsSymbolSearch = _
   private var compilers: Compilers = _
   private var scalafixProvider: ScalafixProvider = _
-  private var fileDecoderProvider: FileDecoderProvider = _
+  var fileDecoderProvider: FileDecoderProvider = _
```

Comment:
Do we need to make it non private? We could just send a command instead.

Reply:
Same as below, could be wait with it until next PR? I have some idea how to improve it, but I would rather do it after bugfixes/improvements.